src/Components/Notification.vue

```diff
@@ -290,6 +290,13 @@ export default {
 		}
 	},
 
+	beforeDestroy() {
+		const event = {
+			notification: this.$props,
+		}
+		emit('notifications:notification:dismissed', event)
```

Comment:
but it might not be dismissed, but just faded out at the end, because there are 25 newer notifications :speak_no_evil: